Header header
uint8 AUTO=0
uint8 MANUAL=1
uint8 START=2
uint8 state
